src/churn/churn.types.ts

```diff
@@ -8,4 +8,12 @@ export interface ChurnSignallingMessage {
   webrtcMessage ?:signals.Message;
   publicEndpoint ?:net.Endpoint;
   caesar ?:number;
+  obfuscator ?:ObfuscatorConfig;
+}
+
+export interface ObfuscatorConfig {
```

Comment:
This creates a naming conflict: sometimes it's a transformer, and sometimes it's an obfuscator.  Let's pick one.